front/components/items/Appointment/helper.ts

```diff
@@ -41,3 +46,21 @@ export const findMatchingAppointment = ({
             return item;
         }
     });
+
+export const setFirstAvaibleAppointmentDate = ({
+    chosenTherapist,
+    dataAppointmentWithTherapist,
+}: propsOfSetFirstAvaibleAppointmentDate) => {
+    if (dataAppointmentWithTherapist && chosenTherapist) {
```

Comment:
can change to warden to prevent code nesting -> 
```suggestion
    if (!dataAppointmentWithTherapist || !chosenTherapist) return new Date()
```